packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -16,7 +16,21 @@ import {InvalidEmailAddressError} from './ediscovery-error';
  * @returns {String} oneFlight key which is a composite of the request parameters
  */
 function createOneFlightKey(reportId, options) {
-  return String(reportId + (options && options.offset && options.size ? options.offset + options.size : ''));
+  let key = String(reportId);
```

Comment:
Assume you need `String(..)` to handle `undefined`?

Reply:
Not likely but better to ensure a string is returned